apps/jobbrowser/src/jobbrowser/templates/job_browser.mako

```diff
@@ -3048,6 +3091,22 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
           });
         }
       }
+
+      self.onTableRowClick = function (event, id) {
+        var openInNewTab = event && (event.which === 2 || event.metaKey || event.ctrlKey);
+        var idToOpen = id || self.id();
+        if (idToOpen && idToOpen !== '-') {
```

Comment:
nit: what does the id value `-` represent?

Reply:
No clue haha, it was there in previous checks (line 2076)